crucible-llvm/src/Lang/Crucible/LLVM/SimpleLoopFixpoint.hs

```diff
@@ -0,0 +1,748 @@
+------------------------------------------------------------------------
+-- |
+-- Module           : Lang.Crucible.LLVM.SimpleLoopFixpoint
+-- Description      : Execution feature to compute loop fixpoint
+-- Copyright        : (c) Galois, Inc 2021
+-- License          : BSD3
+-- Stability        : provisional
+------------------------------------------------------------------------
```

Comment:
I'd like to see some general documentation about what this module does and what its limitations are. I know we talked about this in person, but we should really preserve those thoughts for posterity.